src/test/tests/simulation/updateplots.py

```diff
@@ -60,7 +77,10 @@ def step(sim):
         i = i+1
 
     TestText("updateplots%02d"%i, times)
-             
+
+    # Uncomment this when #17008 is fixed (crash when Logging ExportDBRPC)
+    #testExportVTK(sim)
+
```

Comment:
Just curious...another option is to have this test *active* but on the skip list. The only reason I think that could be preferred is that the skip list is the official record of skip tests.

Thoughts?

That said, we should review what is on the skip list with more regularity than we currently do and a `find . -name '*.py' -exec grep -i '^\s*test.*(.*)'` would likely reveal any tests we currently have commented out.

Reply:
Can't have it on the skip list as it is an individual case that causes a crash.  The test itself won't be skipped, the result would simply be tagged as skipped.  The only items in the skip list that are truly not executed are those that skip the entire test file.

Reply:
Ah, ok, good to know. That sounds like an issue I should add to [here](https://github.com/visit-dav/visit/issues/5297)